qemu/tests/qemu_guest_agent.py

```diff
@@ -1828,6 +1828,44 @@ def gagent_check_nonexistent_cmd(self, test, params, env):
         else:
             test.fail("Should return error info.")
 
+    @error_context.context_aware
+    def gagent_check_log(self, test, params, env):
+        """
+        Check guest agent logs.
+        Steps:
+        1) start guest-agent to record logs
+        2) issue some guest agent commands
+        3) check agent log, if those commands are recorded
+
+        :param test: kvm test object
+        :param params: Dictionary with the test parameterspy
+        :param env: Dictionary with test environment.
+        """
+        session = self._get_session(self.params, self.vm)
+        self._open_session_list.append(session)
+        self._change_bl(session)
+
+        error_context.context("Issue some common guest agent commands.",
+                              logging.info)
+        qga_cmd_list = ["guest-get-time", "guest-file-open",
+                        "guest-file-write", "guest-file-read"]
```

Comment:
@xiagao If anyone agent command can be recorded in log.  I think you only need test a agent command is enough(e.g guest_file_read).

Reply:
For linux guest.  looks good to me.

Acked by:Xiangchun fu xfu@redhat.com